languages/csharp/exercises/concept/integral-numbers/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+C#, like many statically typed languages, provides a number of types that represent integers, each with its own range of values. At the low end the `sbyte` type has a minimum value of -128 and a maximum value of 127. Like all the integer types these values are available as `<type>.MinValue` and `<type>.MaxValue`. At the high end the `long` type has a minimum value of -9,223,372,036,854,775,808 and a maximum value of 9,223,372,036,854,775,807. In between lie the `short` and `int` types.
+
+Each of the above types is paired with an unsigned equivalent: `sbyte`/`byte`, `short`/`ushort`, `int`/`uint` and `long`/`ulong`. In all cases the range of the values is from 0 to the negative signed maximum times 2 plus 1.
```

Comment:
Would it perhaps make sense to have some sort of table listing the types and their ranges (and the number of bits/bytes?)

Reply:
closed to finalize - will reopen shortly

Reply:
Done